src/Validator/IsInt.php

```diff
@@ -39,6 +42,13 @@ class IsInt extends AbstractValidator
     protected $locale;
 
     /**
+     * Type of strict check to be used.  "123" == 123
+     *
+     * @var int
+     */
+    protected $strict = self::COMPARE_NOT_STRICT;
```

Comment:
Why would this value be an `int`, rather than `bool`? I can't see there being any other values (strict vs not)